generator/annual-report/sig_report.tmpl

```diff
@@ -46,52 +46,11 @@
   {{ end}}
 {{ end}}
 
+4. Did you have community-wide updates in {{lastYear}} (e.g. KubeCon talks)?
 
-## Project health
-
-1. What areas and/or subprojects does your group need the most help with?
-   Any areas with 2 or fewer OWNERs? (link to more details)
-
-   -
-   -
-   -
-
-2. What metrics/community health stats does your group care about and/or measure?
-
-   -
-   -
-   -
-
-3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
-   to activities or programs that provide useful context or allow easy participation?
-
-   -
-
-4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
-   does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
-
-   -
-
-5. Does the group have contributors from multiple companies/affiliations?
-
-   -
-
-6. Are there ways end users/companies can contribute that they currently are not?
-   If one of those ways is more full time support, what would they work on and why?
-
-   -
-   -
-
-## Membership
-
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-
-Include any other ways you measure group membership
+<!--
+  Examples include links to email, slides, or recordings.
+-->
```

Comment:
I think for this we should move it above the generated content, that way its a clear thing between user supplied content and generated content.